examples/example_O2_PES_PAD.py

```diff
@@ -117,7 +117,7 @@ def PAD(theta, beta, amp):
                     wspace=0.35, hspace=0.37)
 
 # Save a image of the plot
-plt.savefig("example_O2_PES_PAD.png", dpi=100)
+plt.savefig("output/example_O2_PES_PAD.png", dpi=100)
```

Comment:
Because of the proliferation of files that are generated. The poor user may not know what to run next. I often use the example scripts to determine what parameters to use. They become hidden. An alternative may be to rename the output plots to plot_example_xxx.png. 

I may be alone in this opinion.